.travis.yml

```diff
@@ -10,7 +10,7 @@ env:
     - RUNTIME=ol RUNTIME_VERSION=19.0.0.9
 before_install:
     - echo 'Installing ci.common lib ....'
-    - git clone https://github.com/OpenLiberty/ci.common.git ./ci.common
+    - git clone -b devModeContainers https://github.com/OpenLiberty/ci.common.git ./ci.common
```

Comment:
If this development branch is not going to stay around, reminder to update this to point back to the default branch after these dependencies are finalized and merged. 

Same for appveyor.yml, and same files in ci.maven repo - 4 files in total

Reply:
Reverted